client/src/components/Banner/Banner.stories.js

```diff
@@ -193,4 +194,14 @@ storiesOf("Layout/Banner", module)
         placeholder="Search Trip"
       />
     </Banner>
+  ))
+  .add("Book Trip Banner with Text", () => (
+    <Banner 
+    bannerType= "booktrip"
+    background={Banner__booktrip}>
+      <h3>Hawaii 2020</h3>
+    </Banner>
   ));
+
+
+  // style={{ textAlign: "right" }}
```

Comment:
Can you delete this?

Reply:
@ZakkMan   Commented code has been deleted.